rosetta/text/vw_helpers.py

```diff
@@ -318,21 +318,18 @@ def _set_probabilities(self, topics, predictions):
         Others can be derived and appear as "properties"
         (in the decorator sense).
         """
-        topic_sums = topics.sum()
-        self.pr_topic = topic_sums / topic_sums.sum()
+        self._lambda_word_sums = topics.sum()
+        self.pr_topic = self._lambda_word_sums / self._lambda_word_sums.sum()
 
+        # tokens & topic
         word_sums = topics.sum(axis=1)
         self.pr_token = word_sums / word_sums.sum()
-        self.pr_topic_token = topics / topics.sum().sum()
+        self.pr_token_topic = topics / self._lambda_word_sums.sum()
+        self.pr_token_topic.index.name = 'token'
```

Comment:
Yeah, I actually like the index names, and a column name would be good as well.

I was saying that rather than adding the index name to pr_token_topic, it would make sense if the `topics` dataframe had index/column names.  Then the same names would propagate through to all derived frames.